content/en/docs/components/pipelines/sdk-v2/v2-compatibility.md

```diff
@@ -1,158 +0,0 @@
-+++
-title = "Introducing Kubeflow Pipelines SDK v2"
-description = "Overview of how to get started with Kubeflow Pipelines SDK v2"
-weight = 10
-+++
-
-{{% beta-status
-  feedbacklink="https://github.com/kubeflow/pipelines/issues" %}}
-
-The Kubeflow Pipelines SDK provides a set of Python packages that you can use to specify and run your machine learning (ML) workflow as a pipeline. Version 2 of the SDK adds support for tracking pipeline runs and artifacts using ML Metadata. Starting with Kubeflow Pipelines 1.6, you can build and run pipelines in v2 compatibility mode.
-
```

Comment:
Since user can still use V2 compatible in pre-1.9.0 SDK, and we haven't officially released V2 yet, can we call out the supported SDK versions for V2 compatible, instead of removing the whole page here?

Reply:
If that is the case, should we remove all other pages under `sdk-v2` as well? https://www.kubeflow.org/docs/components/pipelines/sdk-v2/. Deleting only one page but not other pages seems to leave the kubeflow doc in a confusing state.